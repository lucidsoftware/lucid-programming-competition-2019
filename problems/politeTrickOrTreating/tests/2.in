26340 intimidating foul-looking mist pure sucrose
68647 spine-chilling green ogre pumpkin
10090 horrendous double chocolate spirit liquorice
74475 shocking oozing coffin frosting
63020 bloodcurdling triple fudge demon sugar
10956 spooky brain melting hocus pocus fudge
45198 intimidating iced spell sucker
-1778 chilling oozing tomb frosting
32976 chilling fireball zombie pure sugar
80145 alarming poisonous-looking nightmare bars
94538 eerie double chocolate hocus pocus cottoncandy
75059 alarming foul-looking werewolf caramel apple
42923 bloodcurdling fireball howl gummy
63702 creepy glucose exploding spell cottoncandy
22686 intimidating quadruple rasberry potion treat
