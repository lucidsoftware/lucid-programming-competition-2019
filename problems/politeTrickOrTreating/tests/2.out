alarming poisonous-looking nightmare bars
